jena-core/src/test/java/org/apache/jena/test/TestSystemSetup.java

```diff
@@ -35,5 +42,24 @@ public void testRDF11() {
             fail("RDF 1.0 mode enabled in Jena3 test run") ;
     }
 
+    /** This test relies on forking a clean JVM */ 
+    public void testInitFromRDFS() throws IOException, InterruptedException {
+        String separator = System.getProperty("file.separator");
+        String classpath = System.getProperty("java.class.path");
+        String java = System.getProperty("java.home")
+                + separator + "bin" + separator + "java";
```

Comment:
On windows, the java executable will end with ".exe". This should avoid any problem:
```java
import org.apache.commons.lang3.SystemUtils;
        if (SystemUtils.IS_OS_WINDOWS)
            java += ".exe";
```